dsc-core/README.md

```diff
@@ -81,3 +81,11 @@ You can work around this by explicitly importing the module before using the com
 Import-Module "{{pkgPathFor "core/dsc-core"}}/Modules/DscCore"
 Start-DscCore (Join-Path {{pkg.svc_config_path}} firewall.ps1) NewFirewallRule
 ```
+
+# Testing
+
+```
```

Comment:
Hey @mwrock 

This information should be removed and the source policy document should be linked to instead.

Please see: #2678 

Here is an example of how to do this:

```
# Testing

To run tests follow the example in the latest [test standards document](https://github.com/habitat-sh/core-plans/blob/master/docs/dev/policy_documents/standards-for-tests.md).
```

Reply:
Hmm. unfortunately in this case that link would be incorrect.  This is one of the probably 2 (the other being `sqlserver`) plans where you should not test in a local studio. In this case there is no guarantee that the local environment is configured for DSC. The instructions here call for running the test in a Docker container which will guarantee a DSC capable environment.

In general I do prefer testing for core-plans to use the lighter weight local studio as is done in the linked doc but for this plan, I think the `-D` will provide a more reliable testing environment.

Reply:
@mwrock We have to be consistent in the context of CI as the method for how we run our tests so that we don't need to define it for every package. I totally get where you're coming from with the default of the local studio, but I think that given the primary places these are intended to run is CI where we can control our environment and that not everything can run in the local studio, the Docker studio has to be our default for Windows.  I think (and my memory is a bit fuzzy here) there were some differences in how the Linux vs Windows studios are run, as far as artifact caching, and flags available that lead us to the current default. We need to loop back and address that but it's been lower on the burner.

That's a long way of saying: I think we're fine here since this is slightly different than what we've stated elsewhere, and we probably want it to be our default anyways.